docs/core/tutorials/snippets/with-visual-studio/vb/Program.vb

```diff
@@ -4,7 +4,7 @@
         Console.WriteLine(vbCrLf + "What is your name? ")
         Dim name = Console.ReadLine()
         Dim currentDate = DateTime.Now
-        Console.WriteLine($"{vbCrLf}Hello, {name}, on {currentDate:d} at {currentDate:t}")
+        Console.WriteLine($"{Environment.NewLine}Hello, {name}, on {currentDate:d} at {currentDate:t}")
         Console.Write(vbCrLf + "Press any key to exit... ")
```

Comment:
For consistency we should change the other two instances of vbCrLf to Environment.NewLine also.